generate-checksum/README.md

```diff
@@ -0,0 +1,26 @@
+# Generate Checksum action
+
+Generates a 512-bit `sha` hash for each file downloaded from artifacts.
+
+## Documentation
+
+### Inputs
+
+| Input | Description | Required | Default |
+| ------------------------------- | ----------------------------------------------------- | ------------------------- | ------------------------- |
+| `hashfile_name` | The name of the generated checksum file | false | "CHECKSUMS.txt" |
+
+## Usage
+
+```yaml
+name: Action
+on: push
+jobs:
+  build:
+    name: Job
+    runs-on: ubuntu-latest
+    steps:
+      - name: OpenAPI
+        id: openapi
```

Comment:
probably better to use name/id related to the action being performed, i.e. `sha512sum`